src/reference-designs/tko-on-vsphere.md

```diff
@@ -446,13 +446,13 @@ VMware recommends that you deploy Tanzu Kubernetes Grid on non-stretched vSphere
 
 ## Container Registry
 
-The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images to be used in your Kubernetes clusters. There are three main supported installation methods for Harbor:
+The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images to be used in your Kubernetes clusters. There are three officially documented installation methods for Harbor:
```

Comment:
You can install Harbor using one of the following methods:

Reply:
This should be addressed in the latest push